types/node-random-name/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for node-random-name 1.0
+// Project: https://github.com/cscott/node-random-name
+// Definitions by: Ankan Bhattacharya <https://github.com/Ankan002>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+
+declare function random_name(args?: random_name.NodeRandomNameArgs): string;
```

Comment:
Looking upstream, this function is actually called `getName`; that may be preferable.

Reply:
@jakebailey Just updated the code with both the changes

Reply:
Sure, but not this thread's. But, reading their upstream readme, they use this name too, so I guess it's fine.

Reply:
@jakebailey Actually I already implemented the name change.